packages/backend/src/app/database/database-connection.ts

```diff
@@ -46,14 +47,15 @@ const getSslConfig = (): boolean | TlsOptions => {
 }
 
 const getMigrations = () => {
+    const commonMigration = [removeStoreAction1676649852890];
     if (env === 'prod') {
         return [
             FlowAndFileProjectId1674788714498,
             initializeSchema1676238396411,
+            ...commonMigration
         ];
     }
-
-    return [];
+    return commonMigration;
```

Comment:
Again, why add unnecessary complications to development?

Reply:
added a comment